libmy/my_num_to_str.h

```diff
@@ -0,0 +1,31 @@
+#ifndef MY_NUM_TO_STR_H
+#define MY_NUM_TO_STR_H
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+/*
+ * Format bytes as a null-terminated hex string, with each byte represented as two characters.
```

Comment:
The correct term is NUL-terminated (after the ASCII mnemonic NUL for the character with value 0), NULL is different.

Reply:
Whoops. That was my typo which I copy+pasted and then Demian copy+pasted from me.